SessionUIKit/Screens/Settings/SessionProSettings/SessionProPaymentScreen.swift

```diff
@@ -0,0 +1,286 @@
+// Copyright © 2025 Rangeproof Pty Ltd. All rights reserved.
+
+import SwiftUI
+import Lucide
+
+public struct SessionProPaymentScreen: View {
+    @EnvironmentObject var host: HostWrapper
+    @State var currentSelection: Int
+    @State private var isShowingTooltip: Bool = false
+    
+    /// There is an issue on `.onAnyInteraction` of the List and `.onTapGuesture` of the TooltipsIcon. The `.onAnyInteraction` will be called first when tapping the TooltipsIcon to dismiss a tooltip.
+    /// This will result in the tooltip will show again right after it dismissed when tapping the TooltipsIcon. This `suppressUntil` is a workaround to fix this issue.
+    @State var suppressUntil: Date = .distantPast
+
+    let tooltipViewId: String = "SessionProPaymentScreenToolTip" // stringlint:ignore
+    private let coordinateSpaceName: String = "SessionProPaymentScreen" // stringlint:ignore
+    
+    private let viewModel: SessionProPaymentScreenContent.ViewModelType
+    
+    public init(viewModel: SessionProPaymentScreenContent.ViewModelType) {
+        self.viewModel = viewModel
+        if
+            case .update(let currentPlan, _, _, _) = viewModel.dataModel.flow,
+            let indexOfCurrentPlan = viewModel.dataModel.plans.firstIndex(of: currentPlan)
+        {
+            self.currentSelection = indexOfCurrentPlan
+        } else {
+            self.currentSelection = 0
+        }
+    }
+    
+    public var body: some View {
+        GeometryReader { geometry in
+            ScrollView(.vertical, showsIndicators: false) {
+                VStack(spacing: Values.mediumSmallSpacing) {
+                    ListItemLogoWithPro(
+                        info: .init(
+                            style: {
+                                switch viewModel.dataModel.flow {
+                                    case .refund, .cancel: return .disabled
+                                    default: return .normal
+                                }
+                            }(),
+                            state: .success(description: viewModel.dataModel.flow.description)
+                        )
+                    )
+                    if case .purchase = viewModel.dataModel.flow {
```

Comment:
Similar minor comment to earlier - might be slightly more readable as a switch statement?